lib/ui/widgets/haptics_container.dart

```diff
@@ -0,0 +1,25 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class HapticsContainer extends StatefulWidget {
+  final Widget content;
```

Comment:
```suggestion
  final Widget child;
```

To respect the flutter name convention 😄 Could you apply this change everywhere? 😄

Reply:
Fixed, thanks for pointing it up!